src/py/flwr/common/constant.py

```diff
@@ -36,3 +38,13 @@
 MESSAGE_TYPE_GET_PARAMETERS = "get_parameters"
 MESSAGE_TYPE_FIT = "fit"
 MESSAGE_TYPE_EVALUATE = "evaluate"
+
+
+class SType:
+    """Serialisation type."""
+
+    NUMPY = "builtin:numpy"
```

Comment:
Is the idea here to add other static `SType` properties in the future?

For the value, I was thinking we could use the convention of using the fully qualified package/module name of the type this returns. For example `numpy.ndarray` or `torch.Tensor`.

Reply:
Yes, exactly. That's the idea.

Reply:
Sure, I think we can use `numpy.ndarray` to make it more consistent with their names.